src/main/MainScreenContainer.js

```diff
@@ -10,9 +13,38 @@ import {
   getCurrentTypingUsers,
 } from '../chat/chatSelectors';
 import { getAuth } from '../account/accountSelectors';
+import registerGCM from '../api/registerGCM';
 import MainScreen from './MainScreen';
 
 class MainScreenContainer extends React.Component {
+  componentWillMount() {
+    const { auth, saveTokenGCM, doNarrow } = this.props;
+    if (Platform.OS === 'android') {
```

Comment:
Extract that in a separate file, and call it *.android.js with an empty function for *.ios.js